2021-01-12 20:27:56,636 INFO    MainThread:14288 [wandb_setup.py:_flush():70] setting env: {}
2021-01-12 20:27:56,636 INFO    MainThread:14288 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2021-01-12 20:27:56,636 INFO    MainThread:14288 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-12 20:27:56,636 INFO    MainThread:14288 [wandb_init.py:_log_setup():303] Logging user logs to C:\Users\kevin\OneDrive\Documents\8th sem\Federated_Pneumonia\src\client\wandb\run-20210112_202756-325lxzuq\logs\debug.log
2021-01-12 20:27:56,637 INFO    MainThread:14288 [wandb_init.py:_log_setup():304] Logging internal logs to C:\Users\kevin\OneDrive\Documents\8th sem\Federated_Pneumonia\src\client\wandb\run-20210112_202756-325lxzuq\logs\debug-internal.log
2021-01-12 20:27:59,646 INFO    MainThread:14288 [wandb_run.py:_console_start():1237] atexit reg
2021-01-12 20:27:59,647 INFO    MainThread:14288 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.WRAP
2021-01-12 20:27:59,648 INFO    MainThread:14288 [wandb_run.py:_redirect():1135] Wrapping output streams.
2021-01-12 20:27:59,648 INFO    MainThread:14288 [wandb_run.py:_redirect():1151] Redirects installed.
2021-01-12 20:29:12,682 INFO    MainThread:14288 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 1
2021-01-12 20:29:12,682 INFO    MainThread:14288 [wandb_run.py:_restore():1179] restore
2021-01-12 20:29:12,686 INFO    MainThread:14288 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "0553a269dbf44e5fafbbb3935670041c"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 880
      total_bytes: 880
    }
  }
}

2021-01-12 20:29:14,780 INFO    MainThread:14288 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "5c980227a5004724b2a2e6af690aa0a1"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 5722
      total_bytes: 5722
    }
  }
}

2021-01-12 20:29:16,783 INFO    MainThread:14288 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "3fa6b35ee426449388c10f56c7ce7266"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 5722
      total_bytes: 5722
    }
  }
}

2021-01-12 20:29:17,935 INFO    MainThread:14288 [wandb_run.py:_show_summary():1469] rendering summary
2021-01-12 20:29:17,938 INFO    MainThread:14288 [wandb_run.py:_show_history():1507] rendering history
2021-01-12 20:29:17,944 INFO    MainThread:14288 [wandb_run.py:_show_files():1536] logging synced files
2021-01-12 20:29:17,948 INFO    MainThread:14288 [internal.py:handle_exit():138] Internal process exited
